Dshop/apps/products_catalogue/tests.py

```diff
@@ -15,3 +20,31 @@ def test_create_category_with_parent():
     parent_category = Category.objects.create(name='Parent Category', is_active=True)
     child_category = Category.objects.create(name='Child Category', is_active=True, parent=parent_category)
     assert child_category.parent == parent_category
+
+
+# tests for creating XML file for Ceneo
+@pytest.fixture
+def products():
+    category = Category.objects.create(name='Test Category', is_active=True)
+    return [
+        Product.objects.create(name='Telewizor SAMSUNG QE65Q77B 65" QLED 4K 120HZ Tizen TV ', price=3999.00,
+                               full_description='Description 1', category=category),
+        Product.objects.create(name='Zegarek sportowy GARMIN Venu 2 Plus Biały', price=1739.00,
+                               full_description='Description 2', category=category),
+    ]
+
+
+@pytest.mark.django_db
+def test_generate_xml_file_for_ceneo(products):
+    request_factory = RequestFactory()
```

Comment:
Hmm please do not use requestFactory.
Please use built in client in pytest - check here: https://github.com/pymasterspl/Dshop/blob/66a7db0db31483d3fdb7783b934fa9afbef565ac/Dshop/apps/products_catalogue/tests/test_products.py#L29
This will remove dependency on django tests.